synthesis/codon/codon_test.go

```diff
@@ -229,7 +230,7 @@ func TestGetCodonFrequency(t *testing.T) {
 	codonFrequencyHashMap := getCodonFrequency(codonString)
 
 	if len(codonFrequencyHashMap) != 64 {
-		t.Errorf("TestGetCodonFrequency has failed. codonFrequencyHashMap does not contain every codon.")
```

Comment:
this might have been an accident, but I think we should have a complete sentence here